x-pack/legacy/plugins/siem/server/lib/timeline/routes/import_timelines_route.test.ts

```diff
@@ -334,7 +334,10 @@ describe('import timelines', () => {
       const result = server.validate(request);
 
       expect(result.badRequest).toHaveBeenCalledWith(
-        'child "file" fails because ["file" is required]'
+        [
+          'Invalid value undefined supplied to : { file: (ReadableRt & { hapi: { filename: string } }) }/file: (ReadableRt & { hapi: { filename: string } })/0: ReadableRt',
```

Comment:
😅 that's quite the error message ... harder to parse, but more info than before!

Reply:
I see how you mean, I found some example in case as well, please find this comment https://github.com/elastic/kibana/pull/62125#discussion_r408759560